Wikipedia/assets/codemirror/resources/addon/search/search.js

```diff
@@ -372,9 +370,27 @@
       });
     }
   
-    CodeMirror.commands.find = function(cm) {clearSearch(cm); doSearch(cm);};
-    CodeMirror.commands.findNext = function(cm) {clearFocusedMatches(cm); doSearch(cm, false, {next: true});};
-    CodeMirror.commands.findPrev = function(cm) {clearFocusedMatches(cm); doSearch(cm, true, {prev: true});};
+    CodeMirror.commands.find = function(cm) { 
+      cm.operation(function () { 
```

Comment:
I tried something similar to this but it threw the match count off. Seems to work with your version 👍